sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/ShowTablesExec.scala

```diff
@@ -50,8 +49,7 @@ case class ShowTablesExec(
 
   private def isTempView(ident: Identifier, catalog: TableCatalog): Boolean = {
     if (CatalogV2Util.isSessionCatalog(catalog)) {
-      session.sessionState.catalog
-        .isTempView((ident.namespace() :+ ident.name()).toImmutableArraySeq)
+      session.sessionState.catalog.isTempView(ident.namespace() :+ ident.name())
```

Comment:
note: this doesn't compile with scala 2.13 and I added `.toSeq` when forward porting this commit to 4.0 and master.